<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDialog</class>
 <widget class="QDialog" name="ClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>191</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <widget class="QPushButton" name="button_connect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>271</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_address">
      <property name="text">
       <string>Server Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_address">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_port">
      <property name="text">
       <string>Server Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_port">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65534</number>
      </property>
      <property name="value">
       <number>8050</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_session_type">
      <property name="text">
       <string>Session Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="combo_session_type"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_session_config">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icon/settings.png</normaloff>:/icon/settings.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
